dpnp/backend/include/dpnp_iface.hpp

```diff
@@ -869,6 +869,20 @@ INP_DLLEXPORT void dpnp_rng_normal_c(void* result, _DataType mean, _DataType std
 template <typename _DataType>
 INP_DLLEXPORT void dpnp_rng_poisson_c(void* result, double lambda, size_t size);
 
+/**
+ * @ingroup BACKEND_API
+ * @brief math library implementation of random number generator (power distribution)
+ *
+ * @param [in]  size   Number of elements in `result` arrays.
+ *
+ * @param [in]  alpha  Shape of the distribution, alpha.
+ *
+ * @param [out] result Output array.
+ *
+ */
+template <typename _DataType>
+INP_DLLEXPORT void dpnp_rng_power_c(void* result, double alpha, size_t size);
```

Comment:
I think it is a good ide to move all `rng` related definitions from main header to sub-header. Example https://github.com/IntelPython/dpnp/blob/master/dpnp/backend/include/dpnp_iface_fft.hpp

Reply:
Other PR might be use for this